desktop/main.ts

```diff
@@ -404,14 +413,13 @@ const mainWindow = () => {
                         submenu: [
                             {
                                 id: 'back',
-                                role: 'back',
```

Comment:
`back` role doesn't exist, and overall since the menu item has click function it wasn't necessary. See more in [docs](https://www.electronjs.org/docs/latest/api/menu-item#roles)